search-app/docs/api.md

```diff
@@ -1859,11 +1859,11 @@ When `has_granules` is set to "true" or "false", results will be restricted to c
 
     curl "%CMR-ENDPOINT%/collections?has_granules=true"
 
-#### <a name="c-has-granules-or-cwic"></a> Find collections with or without granules, or the collection is tagged with the configured CWIC tag.
+#### <a name="c-has-granules-or-opensearch"></a> Find collections with or without granules, or the collection is tagged with the configured OpenSearch tag.
 
-The `has_granules_or_cwic` parameter can be set to "true" or "false". When true, the results will be restricted to collections with granules or with the configured CWIC tag.  When false, results will be restricted to collections without granules.
+The `has_granules_or_opensearch` parameter can be set to "true" or "false". When true, the results will be restricted to collections with granules or with the configured OpenSearch tag (opensearch.granules.osdd).  When false, results will be restricted to collections without granules.
```

Comment:
Can you clarify `When false, results will be restricted to collections without granules.` in the doc? is that all collections without granules or all collections that are tagged with Opensearch tag and also are without granules?

Reply:
I think it is neither. It is collections not tagged and also without granules.

Reply:
The functionality for `false` is collections which either:
- have no granules, OR 
- have the configured OpenSearch tag

I considered changing this, but I ultimately decided on keeping functionality this way to match `has_granules_or_cwic`. Although, I would not be opposed to changing the functionality above to an AND instead of an OR, if others agree that would be preferable. If not, I will just clarify the docs.